### ISSUE TITLE
Getting invalid schedule for several channels...

### ISSUE THREAD

**Issue by rykr** [kgroeneveld/tv_grab_sd_json#13]
I've tried tv_grab_sd_json and tv_grab_zz_sdjson.  both for tvheadend.    I tried deleting the cache files.   What could be causing this?

**Comment by bonelifer** [comment]
Possibly caused by this?  http://forums.schedulesdirect.org/viewtopic.php?f=18&t=2919

**Comment by rykr** [comment]
Good call. Yes that is probably it.  Will wait for that to clear up and see if the errors disappear.  Thanks
